geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/SqlHandler.java

```diff
@@ -54,6 +60,13 @@ public SqlHandler(InternalCache cache, String regionName,
     this.tableMetaDataManager = tableMetaDataManager;
     this.regionMapping = getMappingForRegion(configService, regionName);
     this.dataSource = getDataSource(dataSourceFactory, this.regionMapping.getDataSourceName());
+    try {
+      compareTableMetadataWithMapping(tableMetaDataManager, regionMapping, regionName);
+    } catch (Exception ex) {
```

Comment:
figure out what you really need to catch. Usually "Exception" can catch more than you intend. Try to make it more specific.

Reply:
Same as above.